docs/extras/web-debugger.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Web Debugger
+sidebar_position: 7
+---
+
+import WebDebugger from '@site/static/web-debugger.png';
+import { CenteredImage } from '@site/src/components/CenteredImage';
+
+# Web Debugger
+
+The Web Debugger is a helpful utility you can embed in your own site when using DevCycle. It allows authorized users to see the current state of their Variables, use Self-Targeting to override those Variables, and see a stream of Events from the SDK as they occur.
+
+The Debugger is a powerful tool for debugging and modifying your DevCycle experience right from the website where it's being used, without
+having to visit the dashboard.
+
+<CenteredImage src={WebDebugger} alt="Screenshot of Web Debugger" height={500} />
+
```

Comment:
Not sure why this isn't leaving a space, but we should add some spacing between the screenshot and the info box